### ISSUE TITLE
twisted.words.test items in dist3.py are not running under Python 3 in tox

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8738]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8738|
|Type|defect|
|Created|2016-08-09 21:16:34Z|
|Branch|https://github.com/twisted/twisted/tree/8738-rodrigc-words-py3|

There are some tests in twisted.words.test which are in dist3.py, but are not
actually running on Python 3 under tox

<details><summary>Searchable metadata</summary>

```
trac-id__8738 8738
type__defect defect
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8738_rodrigc_words_py3 8738-rodrigc-words-py3
branch_author__ 
status__closed closed
resolution__fixed fixed
component__words words
keywords__None None
time__1470777394734471 1470777394734471
changetime__1474198049187875 1474198049187875
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

https://github.com/twisted/twisted/pull/480

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset f42ff53e4057460b0337333fb04200b1a4d8b86e
```
#!CommitTicketReference repository="" revision="f42ff53e4057460b0337333fb04200b1a4d8b86e"
Merge pull request #480 from twisted/8738-rodrigc-words-py3

Author: rodrigc
Reviewer: alex
Fixes: #8738

Add missing twisted.words entries to dist3.py so that Python 3 tests run
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
